helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -172,7 +172,7 @@ public Response instancesOperations(@PathParam("clusterId") String clusterId,
         admin.enableInstance(clusterId, enableInstances, false);
         break;
       case stoppable:
-        return batchGetStoppableInstances(clusterId, node);
+        return batchGetStoppableInstances(clusterId, node, Boolean.valueOf(skipZKRead));
```

Comment:
Nit: since parameter is `boolean` type, `Boolean.valueOf` -> `Boolean.parseBoolean()`?

Reply:
parseBoolean can cause exception if that does not match. But the valueOf has a default value.